contrib/ci/dependencies.xml

```diff
@@ -1553,4 +1553,10 @@
       <package>ShellCheck</package>
     </distro>
   </dependency>
+  <dependency type="build" id="libprotobuf-c">
+    <distro id="ubuntu">
+      <package>libprotobuf-c-dev</package>
+      <package>protobuf-c-compiler</package>
```

Comment:
I split this up a little bit in what I think might help fix it, hope that's ok.

Reply:
Don't mind at all. My Ubuntu 20 dev box crashed, so struggling...protobuf has 2 compilers protoc & protoc-c. Internally protoc invokes protoc-c.